src/xunit.performance.api/ConsoleOutput/XunitPerformanceLogger.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    internal static class XunitPerformanceLogger
```

Comment:
Can you add a class-level comment on what this is used for (and where it is used)?